cpp/src/ray/runtime/task/native_task_submitter.cc

```diff
@@ -52,9 +52,18 @@ ObjectID NativeTaskSubmitter::Submit(InvocationSpec &invocation,
         invocation.actor_id, BuildRayFunction(invocation), invocation.args, options);
   } else {
     BundleID bundle_id = GetBundleID(call_options);
-    return_refs =
-        core_worker.SubmitTask(BuildRayFunction(invocation), invocation.args, options, 1,
-                               false, std::move(bundle_id), true, "");
+    rpc::SchedulingStrategy scheduling_strategy;
+    if (!bundle_id.first.IsNil()) {
+      auto placement_group_scheduling_strategy =
+          scheduling_strategy.mutable_placement_group_scheduling_strategy();
+      placement_group_scheduling_strategy->set_placement_group_id(
+          bundle_id.first.Binary());
+      placement_group_scheduling_strategy->set_placement_group_bundle_index(
+          bundle_id.second);
+      placement_group_scheduling_strategy->set_placement_group_capture_child_tasks(false);
+    }
+    return_refs = core_worker.SubmitTask(BuildRayFunction(invocation), invocation.args,
+                                         options, 1, false, scheduling_strategy, "");
```

Comment:
Could we split out the placement group strategy refactor in a separate PR this can be based on?